arviz/utils.py

```diff
@@ -664,3 +665,34 @@ def credible_interval_warning(credible_interval, hdi_prob):
 
     hdi_prob = credible_interval
     return hdi_prob
+
+
+class HtmlTemplate:
+    html_template = """
+            <div>
+              <div class='xr-header'>
+                <div class="xr-obj-type">arviz.InferenceData</div>
+              </div>
+              {}
+            </div>
+            """
+    element_template = """
+            <ul class="xr-sections">
```

Comment:
Currently the vertical spacing between groups is very large. I think this could be due to `ul` being part of the element template as this is defining a different list for each group. Can you try moving it to the html template so that we have a single list (only one `ul` defined in html template) and the groups are then elements of this single list (one `li` per group)

Reply:
This `ul`/`li` change is also marked as resolved but it is not and again I have a vague memory of seeing the `ul` moved to html instead of element template :/